presentation/tests.py

```diff
@@ -1,3 +1,58 @@
+from django.contrib.auth.models import AnonymousUser
+from django.core.urlresolvers import reverse
 from django.test import TestCase
+from django.test import RequestFactory
+
+from presentation.models import Presentation
+from presentation.views import PresentationList
+from warp.users.models import User
+
+
+class PresentationListTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.test_user = self.create_test_user()
+        self.presentation = Presentation(subject="subject",
```

Comment:
Our Presentation model has no limitation except length. It means that the model should accept any values with any length less than limit length. So, if testcase is failed with random values, the view may have some problems.

And, if something is wrong, you can use debug tools. Random values would not be problems.